### DISCUSSION TITLE
Database Administration option in Pimcore 11

### DISCUSSION THREAD

**Discussion by Amarjeet664** [pimcore/pimcore#15916] [General]
Hi,

It looks like along with other options **_Database Administration_** option is now not visible out of the box in Pimcore 11 under _Tools -> System Info & Tools_.

I already enabled Server file explorer but couldn't found any package to enable Database administration option.

Is it possible to enable the option, If so, how?

**Comment by dvesh3** [comment]
Hi @Amarjeet664 Database Administration has been removed from v11 as it was getting hard to maintain the third party package. Read more details here https://github.com/pimcore/pimcore/issues/11304

sorry, I'm not aware of any bundle that add this feature to Pimcore 11.

**Comment by Amarjeet664** [comment]
okay, thanks @dvesh3 for this info.
It would be great to include Database Administration in near future or in upcoming versions as it is helpful feature.
